src/main/java/de/szut/lf8_project/lf8/project/ProjectMapper.java

```diff
@@ -2,36 +2,68 @@
 
 import de.szut.lf8_project.lf8.project.dto.CreateProjectDto;
 import de.szut.lf8_project.lf8.project.dto.GetProjectDto;
+import de.szut.lf8_project.lf8.timemanagement.TimeManagementEntity;
+import de.szut.lf8_project.lf8.timemanagement.TimeManagementService;
+import de.szut.lf8_project.lf8.timemanagement.dto.GetTimeManagementDto;
+import de.szut.lf8_project.utils.HelperFunctions;
+import lombok.AllArgsConstructor;
 import org.springframework.stereotype.Service;
 
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+
 @Service
+@AllArgsConstructor
 public class ProjectMapper {
+    private TimeManagementService tmService;
+
     public GetProjectDto mapToGetProjectDto(ProjectEntity entity) {
+        Set<GetTimeManagementDto> employees = new HashSet<>();
+
+        for (TimeManagementEntity employee : entity.getEmployees()) {
```

Comment:
I finding the naming here also confusing, because the variable name suggest the variable is of the type employee, but the eployee is of type TimeManagementEntity.